infrastructure/controller/pkg/configmap-controller/create.go

```diff
@@ -0,0 +1,40 @@
+package configmapController
+
+import (
+	"github.com/airyhq/airy/infrastructure/lib/go/k8s/handler"
+	"github.com/airyhq/airy/infrastructure/lib/go/k8s/util"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog"
+)
+
+// ResourceCreatedHandler contains new objects
+type ResourceCreatedHandler struct {
+	ConfigMap   *v1.ConfigMap
+}
+
+// Handle processes the newly created resource
+func (r ResourceCreatedHandler) Handle(clientSet kubernetes.Interface) error {
+	klog.Infof("Added configMap: %s , sha: %s", r.ConfigMap.GetName(), util.GetSHAfromConfigmap(r.ConfigMap))
+	affectedDeployments, errGetDeployments := handler.GetAffectedDeploymentsConfigmap(clientSet,
+		r.ConfigMap.Name, "default", "")
+	if errGetDeployments != nil {
+		klog.Errorf("Error retrieving affected deployments %v", errGetDeployments)
+		return errGetDeployments
+	}
+
+	for _, affectedDeployment := range affectedDeployments {
+		klog.Infof("Scheduling start for deployment: %s", affectedDeployment)
+		if err := handler.ScaleDeployment(handler.ScaleCommand{
+			ClientSet: clientSet,
+			Namespace: "default",
```

Comment:
I'm not sure what's the best solution here... but as I understand it this wouldn't work if one would namespace their core installation?

Reply:
True, I will make it so that it can read this from the configuration. Or do you think it should pick up the default from the config file?

Reply:
I like the ENV idea as discussed offline 👍